src/client/pages/OfferNew/Checkout/UserDetailsForm.tsx

```diff
@@ -114,15 +124,9 @@ export const UserDetailsForm: React.FC<Props> = ({
           pattern="[0-9]*"
           maxLength={ssnMaxLength}
           value={ssn}
-          // errors={ssnError ? textKeys.SIGN_SSN_CHECK() : undefined} TODO error handling?
+          errors={ssnBackendError ? textKeys[ssnBackendError]() : undefined}
```

Comment:
i'd like to challenge whether we should use the error code from backend and translate that into a text key.. Could we maybe have some kind of mapping, and maybe map unknown errors to some generic text key (for example, if backend starts giving an error we don't have a text key for yet)

Reply:
It's so few in this case but as you say it can be hard to keep up if this changes, so good point! I'll have a look!